ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -290,21 +303,40 @@ impl Mutation {
         ctx.locator.auth().refresh_token(refresh_token).await
     }
 
-    async fn create_invitation(ctx: &Context, email: String) -> Result<i32> {
+    async fn create_invitation(ctx: &Context, email: String) -> Result<ID> {
```

Comment:
client side doesn't consumed the returned id, so we can change the return type directly (without creating a new create_invitation_next).